File: Day_2/bmi.cpp
/*
   Day 2: BMI Calculator

   This is a program to calculate a user's BMI from either
   imperial or metric units.
*/
#include <iostream>
using namespace std;

int main()
{
   int choice;
   float weight;
   float height;

   // Note: No error checking yet (Loops tomorrow)
   cout << "Choose your units\n\t1.Metric\n\t2.Imperial\nChoice: ";
   cin >> choice;
   cout << endl;

   // Calculate the BMI with metric units
   if(choice == 1)
   {
      cout << "Enter your height (m): ";
      cin >> height;
      cout << "\nEnter your weight (kg): ";
      cin >> weight;
      cout << "\nBMI: " << weight / (height * height) << endl;
   }

   // Calculate BMI with imperial units
   else
   {
      cout << "Enter your height (in): ";
      cin >> height;
      cout << "\nEnter your weight (lbs): ";
      cin >> weight;
      cout << "\nBMI: " << 703 * weight / (height * height) << endl;
   }

   return 0;
}

File: Day_2/bmi.py
"""
   Day 2: BMI Calculator

   This is a program to calculate a user's BMI from either
   imperial or metric units.
"""

choice = input("Choose your units\n\t1.Metric\n\t2.Imperial\nChoice: ")

# Calculate the BMI with metric units
if choice == str(1):
   height = float(input("\nEnter your height (m): "))
   weight = float(input("\nEnter your weight (kg): "))
   print("BMI: " + str(weight / (height * height)))
# Calculate BMI with imperial units
else:
   height = float(input("\nEnter your height (in): "))
   weight = float(input("\nEnter your weight (lbs): "))
   print("BMI: " + str(703.0 * weight / (height * height)))
